infrastructure/sentry-subgraph/src/node-license.ts

```diff
@@ -23,8 +23,17 @@ export function handleTransfer(event: TransferEvent): void {
   
   sentryWallet.keyCount = sentryWallet.keyCount.plus(BigInt.fromI32(1))
   sentryWallet.save();
-
-  //TODO should keys ever be transferable we need to decrease the keyCount of the from address
+  
+  //Decrease the keyCount of the from address if not a mint event
+  if (event.params.from != Address.zero()) {
+    const fromSentryWallet = SentryWallet.load(event.params.from.toHexString());
+    if (fromSentryWallet) {
+      fromSentryWallet.keyCount = fromSentryWallet.keyCount.minus(
+        BigInt.fromI32(1)
+      );
+      fromSentryWallet.save();
+    }
```

Comment:
If we don't have the sentry wallet we should log an error like we doe in the other null cases.

Reply:
Done, I was assuming a Sentry Wallet should always be present as it should have been created upon receipt of the key, but we now log a warning if that's not the case.